### ISSUE TITLE
Table - allow providing aria-label to the input for row selection

### ISSUE THREAD

**Issue by dgdavid** [patternfly/patternfly-react#11912]
Hi there 👋

Just wondering if it would make sense to allow providing a custom aria-label for the input used for row selection (radio or checkbox, depending on the [TdSelectType](https://www.patternfly.org/components/table#tdselecttype) variant). From what I can see, the label is currently hardcoded as `Select row ${rowIndex}` in the [selectable decorator](https://github.com/patternfly/patternfly-react/blob/87ddd2d8cf02bacbf69c8950daff0137a9fc5c74/packages/react-table/src/components/Table/utils/decorators/selectable.tsx#L41), and I couldn't find a way to customize it.

I'm not sure if I'm missing something or if overriding that label would go against intended usage, but I believe being able to provide a custom value to at least support translations might be useful in some cases.

Would this be a reasonable thing to support?

Thanks!

**Comment by thatblindgeye** [comment]
Hey @dgdavid . Yeah I think that could be helpful - referencing the "primary" data cell in a row might be more helpful to a user ("Select Cluster Zed row" vs "Select row 1" for example), plus for internationalization purposes.

I'll update this issue a bit to triage it properly - if you'd be interested in working on this just let me know
